### DISCUSSION TITLE
Rotate Touchscreen

### DISCUSSION THREAD

**Discussion by f1nch87** [debloper/piosk#60] [Q&A] [answered]
Hey,
I like the project but the thing im struggeling the most while manually setup kiosk is to rotate a touchscreen image and touch under wayland.

Is there a way to implement it here maybe?

**Comment by debloper** [comment, marked as answer]
Thanks for using PiOSK.

As for screen rotation; I think for now it's best to leave it on the OS (desktop manager) or the browser API to figure it out. It's one of those things, that looks trivial but isn't (and opens a pandora's box of bugs & edge cases). Given the desktop managers and browsers literally spent ages implementing these features; it's unlikely that we'll do a better job than them, on a shoestring.

In short, it's out of scope for now.

**Reply by f1nch87** [reply]
Thx for ur answer.
I understand you. I had spent some hours on it without success. I use screen management pi tool now and it seemlessly working with labwc.
